bin/run-tests-in-docker.sh

```diff
@@ -15,17 +15,16 @@
 # Stop executing when a command returns a non-zero return code
 set -e
 
-# Build the Docker image
+# Build the Docker image  
 docker build --rm -t exercism/test-runner .
 
 # Run the Docker image using the settings mimicking the production environment
 docker run \
     --rm \
     --network none \
     --read-only \
-    --mount type=bind,src="${PWD}/tests",dst=/opt/test-runner/tests \
+    --mount type=bind,src="${PWD}",dst=/opt/test-runner/ \
```

Comment:
Was there a particular reason that you need to map the full directory instead of just tests? Just curious

Reply:
I'll check again, but I'm fairly sure that's because Pyret creates a mandatory .pyret folder containing the compiled .arr files in the current working directory when you run the `pyret` command. We can't set an alternative location so I think we could just change the working directory to the base of the tests subfolder in the scripts.

Reply:
Yeah, if I revert back to the generic Docker command, the Docker image works well locally. In the CI workflow, [it can't create the .pyret folder](https://github.com/BNAndras/exercism-pyret-test-runner/actions/runs/5677151653/job/15384993530?pr=3) at the base of test-runner. If I change the current working directory to each test's folder, the .pyret folder should be able to be created. However, [it hangs](https://github.com/BNAndras/exercism-pyret-test-runner/actions/runs/5677304478/job/15385440420) on the first test encountered so that's no good.

Reply:
Okay, we can ignore my comment then!